ethpm/assets/package.schema.v2.json

```diff
@@ -182,10 +188,11 @@
 	  "title": "Contract Bytecode",
 	  "type": "object",
 	  "required": ["bytecode"],
+	  "additionalProperties": false,
 	  "properties": {
 		"bytecode": {
 		  "type": "string",
-		  "pattern": "^0x[0-9a-fA-Ffff]*$"
+		  "pattern": "^0x[0-9a-zA-Z_]*$"
```

Comment:
Made this change since bytecode will have contract names in them? which would require all the chars. Though this might change as `ethpm-spec` #70 gets finalized

Reply:
I would just make it conform to https://github.com/ethpm/ethpm-spec/pull/70 since the old format is going away and fundamentally broken for certain cases.